src/sentry/relay/config.py

```diff
@@ -162,15 +162,12 @@ def get_project_config(project, full_config=True, project_keys=None):
         if dynamic_sampling is not None:
             cfg["config"]["dynamicSampling"] = dynamic_sampling
 
-    if features.has("organizations:performance-ops-breakdown", project.organization):
-        breakdowns_config = project.get_option("sentry:breakdowns")
-        if breakdowns_config is not None:
-            cfg["config"]["breakdowns"] = breakdowns_config
-
     if not full_config:
         # This is all we need for external Relay processors
         return ProjectConfig(project, **cfg)
 
+    if features.has("organizations:performance-ops-breakdown", project.organization):
+        cfg["config"]["breakdowns"] = project.get_option("sentry:breakdowns")
```

Comment:
@jan-auer @untitaker Just for my understanding, what is considered trusted Relays?

Reply:
Does this new placement ensure this breakdown config will only go out to SaaS Relays?

Reply:
yes exactly and that's also definition of trusted relays

idk where pop relays stand, but it shouldn't matter for this PR

Reply:
Trusted Relays are meant to handle traffic across all organizations within our infrastructure, which comprises PoPs and processing Relays.